pkg/ingester/ingester_v2.go

```diff
@@ -992,6 +992,11 @@ func (i *Ingester) v2Query(ctx context.Context, req *client.QueryRequest) (*clie
 
 	i.metrics.queries.Inc()
 
+	if i.State() == services.Stopping {
+		level.Warn(i.logger).Log("Cannot retrieve TSDB, as the Ingester is in the process of stopping and closing all TSBD")
```

Comment:
Logs must be in the form `("msg", "<put the message here>")`. Also `TSDB` is mispelled.

```suggestion
		level.Warn(i.logger).Log("msg", "Cannot retrieve TSDB, as the Ingester is in the process of stopping and closing all TSDB")
```

_This comment applies to all other places too._

Reply:
Also, is this a warning? Doesn't like something actionable. It's by design, so I'm wondering if we should log it at all (or a debug at most). @pstibrany ?

Reply:
I changed it to a debug level, but if it isn't required, I can remove the log message completely.

Reply:
I don't think this log message is too helpful. Cortex already logs message when it starts its shutdown procedure.

Reply:
That makes sense. I've removed the log message.